app/build.gradle

```diff
@@ -1,30 +1,41 @@
-/*
- * This file was generated by the Gradle 'init' task.
- *
- * This generated file contains a sample Java application project to get you started.
- * For more details take a look at the 'Building Java & JVM projects' chapter in the Gradle
- * User Manual available at https://docs.gradle.org/7.4.2/userguide/building_java_projects.html
- */
-
 plugins {
-    // Apply the application plugin to add support for building a CLI application in Java.
     id 'application'
+    id 'checkstyle'
+    id 'com.adarshr.test-logger' version '3.2.0'
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(17)
+    }
 }
 
 repositories {
-    // Use Maven Central for resolving dependencies.
     mavenCentral()
 }
 
 dependencies {
-    // Use JUnit test framework.
     testImplementation 'junit:junit:4.13.2'
-
-    // This dependency is used by the application.
     implementation 'com.google.guava:guava:30.1.1-jre'
 }
 
+test {
+    useJUnit()
+}
+
 application {
-    // Define the main class for the application.
-    mainClass = 'echoserver.App'
+    mainClass = 'EchoServer.App'
```

Comment:
Do we want this change? Maybe it's just me, but I'm not able to run the app unless I use the previous change

<img width="822" alt="Screen Shot 2022-07-05 at 8 19 22 AM" src="https://user-images.githubusercontent.com/6445410/177362496-f63cf54e-dab5-4364-b31b-75068921e454.png">